kubernetes/istio_details_service.go

```diff
@@ -155,6 +158,7 @@ func (in *IstioClient) getRbacResources() map[string]bool {
 	}
 
 	rbacResources := map[string]bool{}
+	// TODO Update with Maistra API
```

Comment:
Is it to do within this PR ?

Reply:
Yeah. I'll update that part.

Reply:
I've reviewed and updated the code.
Basically the comment was not correct as the right logic to check if it needs to get a ClusterRbacConfig or a ServiceMeshRbacConfig is handled in the upper methods "GetIstioConfigList" and "GetAuthorizationDetails".